packages/sources/src/components/definition-list/definition-list.stories.ts

```diff
@@ -140,4 +140,14 @@ export function storiesOfDefinitionList<TemplateFnReturnType>(
       }
     }
   );
+
+  stories.add(
+    'bordered',
+    template,
+    {
+      args: {
+        modifier: 'dso-bordered'
```

Comment:
In het issue is toch echt wel heel expliciet benoemd dat de colon (`:`) niet visueel aanwezig moet zijn. Dus ik denk dat we er niet aan ontkomen om dat:
- in de njk templating als parameter te kunnen instellen (iets als `useSrOnlyColon`)
- ook in de ts templating